ios/NotifeeCore/NotifeeCore.m

```diff
@@ -178,14 +179,32 @@ + (void)displayNotification:(NSDictionary *)notification withBlock:(notifeeMetho
   UNMutableNotificationContent *content = [self buildNotificationContent:notification
                                                              withTrigger:nil];
 
-  UNNotificationRequest *request = [UNNotificationRequest requestWithIdentifier:notification[@"id"]
-                                                                        content:content
-                                                                        trigger:nil];
   UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
 
   NSMutableDictionary *notificationDetail = [notification mutableCopy];
   notificationDetail[@"remote"] = @NO;
 
+  if (@available(iOS 15.0, *)) {
```

Comment:
just for completeness I was looking at where this was implemented and what the iOS15 restriction was, whether other platforms supported it etc, looks like it *may* be clamped to 15 with interfaces like this, but it is on other platforms? https://developer.apple.com/documentation/sirikit/insendmessageintentdonationmetadata?language=objc

Reply:
good point! I will update this with the other platforms (in another p/r) 😎